chatbot.js

```diff
@@ -46,6 +47,8 @@ const getBotReply = (username, message, parsedMessage) => {
     places.getPlaces(api.postBotReply, username, cityName);
   } else if (startsWithString === constants.BOT_ACTIONS.DEFINE) {
     define.define(api.postBotReply, username, cityName);
+  } else if (startsWithString === constants.BOT_ACTIONS.INSULT) {
+    insult.insult(api.postBotReply, cityName);
```

Comment:
1. `insulter.getInsult` -> Rename
2. Shouldn't you be passing the `username` instead of `cityName`